RDKEMW-8862: Fixing Coding Guideline Errors

kkarup694 opened the pull request:
Reason for change: resolving guidelines errors
Test Procedure: Mentioned in ticket
Risks: Low

kkarup694 committed:
RDKEMW-8862: Fixing Coding Guideline Errors

Reason for change: resolving guidelines errors
Test Procedure: Mentioned in ticket
Risks: Low

Signed-off-by: kumar <kumar_Karuppiahyogarajah@comcast.com>

scthunderbolt commented:
Build error
"
[  7%] Building CXX object ttsclient/CMakeFiles/TextToSpeechServiceClient.dir/TextToSpeechServiceCOMRPC.cpp.o
In file included from /root/install/usr/include/interfaces/ITextToSpeech.h:23,
                 from /root/ttsengine/ttsclient/TextToSpeechServiceCOMRPC.h:51,
                 from /root/ttsengine/ttsclient/TextToSpeechServiceCOMRPC.cpp:20:
/root/entservices-apis/apis/Module.h:39:10: fatal error: interfaces/entservices_errorcodes.h: No such file or directory
   39 | #include <interfaces/entservices_errorcodes.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
"

kkarup694 committed:
test

kkarup694 committed:
RDKEMW-8862: fixing coding guidelines

Reason for change: need to address coding guidelines in TTS
Test Procedure: Mentioned in ticket
Risks: Low

Signed-off-by: kumar <kumar_Karuppiahyogarajah@comcast.com>

kkarup694 committed:
RDKEMW-8862: fixing coding guidelines

Reason for change: need to address coding guidelines in TTS
Test Procedure: Mentioned in ticket
Risks: Low

Signed-off-by: kumar <kumar_Karuppiahyogarajah@comcast.com>

kkarup694 committed:
RDKEMW-8862: fixing coding guidelines

Reason for change: need to address coding guidelines in TTS
Test Procedure: Mentioned in ticket
Risks: Low

Signed-off-by: kumar <kumar_Karuppiahyogarajah@comcast.com>
